activesupport/lib/active_support/core_ext/numeric/conversions.rb

```diff
@@ -97,7 +97,10 @@ class Numeric
   #  1234567.to_s(:human, precision: 1,
   #                   separator: ',',
   #                   significant: false)                   # => "1,2 Million"
-  def to_formatted_s(format = :default, options = {})
+  def to_s(*args)
```

Comment:
Is there no way we can make it work without resorting to splatting? Would `format = :default, **options` work?